debian/patches/skip_support_file_installation.patch

```diff
@@ -0,0 +1,34 @@
+From: Carlos Maddela <e7appew@gmail.com>
+Date: Tue, 1 Mar 2016 02:31:47 +1100
+Subject: Don't install support files via setup.py
+
+Description: Don't install support files via setup.py
+ for better control over where they should be installed.
+Author: Carlos Maddela <e7appew@gmail.com>
+Origin: vendor
+Forwarded: not-needed
+Last-Update: 2016-06-27
```

Comment:
Great, thanks. If they have not yet made their "bulid system" more flexible, perhaps we should ask them to, so that we can make use of that, instead of patching every new release of theirs.